package.json

```diff
@@ -24,6 +24,7 @@
     "which": "^1.0.9"
   },
   "scripts": {
+    "preinstall": "npm install tsd && tsd update",
```

Comment:
ah, neat. Good point that travis would have been broken if this were wrong. Thanks!